libs/application/templates/new-primary-school/src/fields/Review/review-groups/Languages.tsx

```
@@ -23,132 +30,107 @@ export const Languages = ({
-                      newPrimarySchoolMessages.differentNeeds
-                        .languageSelectionTitle,
-                      { no: `${3}` },
-                    )}
-                    value={getLanguageByCode(language3)?.name}
-                  />
-                </GridColumn>
+          ) : (
+            <>
+              <GridRow rowGap={2}>
+                {selectedLanguages?.map(({ code }, index) => (
+                  <GridColumn span={['12/12', '5/12', '5/12', '5/12']}>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)